app/detail mock.html

```
@@ -0,0 +1,139 @@
+                                                        class="text-[10px] font-bold text-gray-400 uppercase tracking-tight">Due
+                                                        Date</span>
+                                                <span class="text-sm font-medium text-gray-700">Oct 26, 2023</span>
+                                        </div>
+                                </div>
+                        </div>
+
+                </div>
+        </main>
+        <div class="p-6 bg-white border-t border-gray-100">
+                <button
+                        class="w-full bg-[#2563eb] text-white font-semibold py-4 rounded-xl shadow-lg shadow-blue-200 active:scale-[0.98] transition-transform">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)